neuraxle/data_container.py

```diff
@@ -142,44 +145,90 @@ def hash_summary(self):
             m.update(str.encode(str(current_id)))
         return m.hexdigest()
 
-    def convolved_1d(self, stride, kernel_size) -> Iterable['DataContainer']:
+    def batch(
```

Comment:
I would prefer this name. 
```suggestion
    def minibatches(
```

Reply:
(because we truly in fact are creating mini batches out of a full batch, where "batch_size" stands for the size of the minibatches. )

Reply:
I saw the batch method in multiple apis. It seemed a lot more familiar and neat to me, but ok